src/main/webapp/app/detail-overview-list/components/link-detail.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, Input } from '@angular/core';
+import type { LinkDetail } from 'app/detail-overview-list/detail.model';
+import { NoDataComponent } from 'app/shared/no-data-component';
+import { RouterModule } from '@angular/router';
+
+@Component({
+    selector: 'jhi-link-detail',
+    template: `
+        <dd id="detail-value-{{ detail.title }}">
```

Comment:
I would rather add this to a separate html-template file to ensure better separation of concerns and maintainability.

Reply:
It was an active decision to move the HTML to the typescript file, since the detail components do not contain actual logic in the typescript, so adding it to a template file would only mean an additional navigation step to me when working with on the details components.

I can change it for this file and future components of a similar template size, do you think it should be changed for the `boolean-detail.component.ts` aswell?

Reply:
Discussed: we want to ensure consistency and use template files regardless of the size of the HTML